playbooks/network.yml

```diff
@@ -15,7 +17,7 @@
           state: started
           enabled: yes
           name: openvswitch
-      when: network_role == 'network-multus' and deploy_ovs_plugin and cluster == "kubernetes"
+      when: network_role == 'network-multus' and deploy_ovs_plugin and platform == "kubernetes"
```

Comment:
Seems if we set `deploy_ovs_plugin: true` in var, then we need to use `when: network_role == 'network-multus' and ( deploy_ovs_plugin | bool ) and platform == "kubernetes"` , or else it will keep true even we set it to false.

Reply:
Done